bitcoin/src/pow.rs

```diff
@@ -1795,8 +1795,8 @@ mod tests {
         // Block 2015, the only information used are `bits` and `time`
         let current = Header {
             version: Version::ONE,
-            prev_blockhash: BlockHash::all_zeros(),
-            merkle_root: TxMerkleNode::all_zeros(),
+            prev_blockhash: BlockHash::from_byte_array([0; 32]),
```

Comment:
`BlockHash` still does have `all_zeros` though, why not call it here?

Reply:
Oh, oops, I must've originally decided not to have `BlockHash::all_zeros` and then reconsidered.

I'll rebase to fix this, sice I want to rebase on the fixed CI anyway.